spec/serializers/person_serializer_spec.rb

```diff
@@ -134,7 +132,7 @@
       end
 
       it 'does not contain an included ethnicity' do
-        expect(result[:included]).to be_nil
+        expect(result[:included]).to be_empty
```

Comment:
This is different behaviour?

Reply:
Yes, it returns an empty array now when there are no includes. I don't think that's a big problem but maybe worth keeping an eye on when testing.

Reply:
Yeah. Depending on how clients are coded this may be impactful. Would be good to have a full UAT session with both suppliers to make sure.

Reply:
I explored this further - it only returns an empty `included` array when the `include` option was passed to the serializer but no valid resources were found to include. In this case we're requesting to include `ethnicity` but then explicitly setting the person ethnicity to `nil`. The output of an empty `included` array seems reasonable in that case, but is a change in behaviour from AMS.